etc/kayobe/ansible/wazuh-manager.yml

```diff
@@ -9,7 +9,6 @@
     - generate-certs
 # Single node
 - hosts: wazuh-manager
-  become: yes
```

Comment:
I trust that the wazuh-ansible roles are happy with this? If so a nice improvement.

Reply:
I haven't tested from a fresh install, but didn't hit any issues when I ran through this to test the agent changes

Reply:
Given a lot is done in the roles, perhaps this should be tested in a multinode...

Reply:
It's definitely worth testing. If you want to merge this soon it might be better to revert to using become and remove it in a separate PR. That would be cleaner than conflating these things anyway.

Reply:
Done